ONSdigital/census-rm-ops
# Census initial contact letter/Sample spec

## Note this PR depends on the action and actionexporter service changes for the changes to work

# Motivation and Context
The sample file spec has changed, as has the print file spec we are expected to produce. This PR updates the sample loader code to match the changes in https://github.com/ONSdigital/census-rm-sample-loader/pull/3, adds the new `ICL1E` action type and removes all other action types as they are now unused by census.

# What has changed
- Replicate sample loader code changes and tests
- Add ICL1E action type
- Remove all other action types
- Change docker image tag in makefile to match that used in census-rm-docker-dev and our GCR

# How to test?
Run with all the sample file/print file spec change branches in docker dev, you should be able to load a new spec sample file and produce an ICL1E print file.

# Links
https://trello.com/c/1oMr4F5D/613-update-census-rm-ops-tool-for-new-sample-spec
https://trello.com/c/e9iWXgje/517-rmc-83-load-hh-sample-and-generate-print-file-to-census-spec-13
https://github.com/ONSdigital/census-rm-sample-loader/pull/3
https://github.com/ONSdigital/census-rm-action-service/pull/3
